src/operator/slice_channel-inl.h

```diff
@@ -176,16 +177,22 @@ class SliceChannelProp : public OperatorProperty {
   bool InferType(std::vector<int> *in_type,
                  std::vector<int> *out_type,
                  std::vector<int> *aux_type) const override {
+    /*
```

Comment:
If this is to be removed, why is it commented? Or is this temporary?

Reply:
yes, it should have been removed. Good catch, fixed.